service/assessment/assessment.go

```diff
@@ -81,6 +123,16 @@ func (s Service) AssessEvidence(_ context.Context, req *assessment.AssessEvidenc
 
 		return res, status.Errorf(codes.InvalidArgument, "invalid req: %v", newError)
 	}
+	// Set up stream to Evidence Store.
+	// TODO(lebogg): If assessment is used as standalone service (without fwd evidences) maybe adapt code, i.e. no error when ConfigureConnections sets no evidence store
+	if s.evidenceStoreStream == nil {
+		if err = s.setEvidenceStoreStream(); err != nil {
+			return nil, fmt.Errorf("could not assess evidence: %v", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("could not set evidence store stream: %v", err)
```

Reply:
Your proposed message is included in the underlying error message and, thus, appended to this error message.

Reply:
Ok, but at this point it is not an "assess evidence" error but an 'could not set stream'-error,  right? 

Shouldn't that be something like `status.Errorf(codes.Internal, "could not set up stream for storing evidences: %v", err)`